flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/internals/publisher/fanout/FanOutRecordPublisherConfiguration.java

```diff
@@ -0,0 +1,433 @@
+	 */
+	@Nullable
+	private String consumerName;
+
+	/**
+	 * The manual set efo consumer arns for each stream. Should not be Null if the efoRegistrationType is NONE
+	 */
+	@Nullable
+	private Map<String, String> streamConsumerArns;
+
+	/**
+	 * Base backoff millis for the deregister stream operation.
+	 */
+	private final int subscribeToShardMaxRetries;
+
+	/**
+	 * Maximum backoff millis for the subscribe to shard operation.
+	 */
+	private final long subscribeToShardMaxBackoffMillis;
+
+	/**
+	 * Base backoff millis for the subscribe to shard operation.
+	 */
+	private final long subscribeToShardBaseBackoffMillis;
+
+	/**
+	 * Exponential backoff power constant for the subscribe to shard operation.
+	 */
+	private final double subscribeToShardExpConstant;
+
+	/**
+	 * Base backoff millis for the register stream operation.
+	 */
+	private final long registerStreamBaseBackoffMillis;
+
+	/**
+	 * Maximum backoff millis for the register stream operation.
+	 */
+	private final long registerStreamMaxBackoffMillis;
+
+	/**
+	 * Exponential backoff power constant for the register stream operation.
+	 */
+	private final double registerStreamExpConstant;
+
+	/**
+	 * Maximum retry attempts for the register stream operation.
+	 */
+	private final int registerStreamMaxRetries;
+
+	/**
+	 * Base backoff millis for the deregister stream operation.
+	 */
+	private final long deregisterStreamBaseBackoffMillis;
+
+	/**
+	 * Maximum backoff millis for the deregister stream operation.
+	 */
+	private final long deregisterStreamMaxBackoffMillis;
+
+	/**
+	 * Exponential backoff power constant for the deregister stream operation.
+	 */
+	private final double deregisterStreamExpConstant;
+
+	/**
+	 * Maximum retry attempts for the deregister stream operation.
+	 */
+	private final int deregisterStreamMaxRetries;
+
+	/**
+	 * Max retries for the describe stream operation.
+	 */
+	private final int describeStreamMaxRetries;
+
+	/**
+	 * Backoff millis for the describe stream operation.
+	 */
+	private final long describeStreamBaseBackoffMillis;
+
+	/**
+	 * Maximum backoff millis for the describe stream operation.
+	 */
+	private final long describeStreamMaxBackoffMillis;
+
+	/**
+	 * Exponential backoff power constant for the describe stream operation.
+	 */
+	private final double describeStreamExpConstant;
+
+	/**
+	 * Max retries for the describe stream consumer operation.
+	 */
+	private final int describeStreamConsumerMaxRetries;
+
+	/**
+	 * Backoff millis for the describe stream consumer operation.
+	 */
+	private final long describeStreamConsumerBaseBackoffMillis;
+
+	/**
+	 * Maximum backoff millis for the describe stream consumer operation.
+	 */
+	private final long describeStreamConsumerMaxBackoffMillis;
+
+	/**
+	 * Exponential backoff power constant for the describe stream consumer operation.
+	 */
+	private final double describeStreamConsumerExpConstant;
+
+	/**
+	 * Creates a FanOutProperties.
```

Comment:
```suggestion
	 * Creates a FanOutRecordPublisherConfiguration.
```

Reply:
Thanks! I've forgot to modify the errors, thanks a lot for helping me!